_data/blogs.yml

```diff
@@ -222,3 +222,8 @@
   title: Flying with Fossasia
   img: arjun.jpg
   author_name: Arjun Pandey
+
+- url:https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title:Here I come Coding In
```

Comment:
please the exact format as given in the readme.md there should be a space after `:`